registrations/tasks.py

```diff
@@ -525,10 +525,15 @@ def create_registration(self, line, source):
 
             if receiver in ('father_only', 'family_only', 'friend_only'):
                 mother_identity = self.get_or_create_identity(
-                    None, receiver_identity['id'])
+                    None, receiver_identity['id'], identity_details)
                 reg_info['mother_id'] = mother_identity['id']
 
-        if mother_identity:
+                receiver_details["linked_to"] = mother_identity['id']
+                utils.patch_identity(receiver_identity['id'],
+                                     {'details': receiver_details})
+
+        if (mother_identity and
```

Comment:
Is this necessary? Won't `mother_identity` always be populated since it's created at the start of the method? (L469)